### DISCUSSION TITLE
Vote for Realtime Connectors

### DISCUSSION THREAD

**Discussion by kiwicopple** [supabase/supabase#4] [General]
This issue is for tracking requests/demand for connectors. Give a comment a thumbs up if you want the connector built. 

Please refrain from adding comments unless you are adding a connector which isn't already listed.

We have not yet implemented the connectors, but here is a prototype of how they may function:

![image](https://user-images.githubusercontent.com/10214025/94219126-a58cfd80-ff18-11ea-87fb-312fce825e3e.png)

**Comment by kiwicopple** [comment]
Kafka

**Comment by kiwicopple** [comment]
SQS

**Comment by kiwicopple** [comment]
RabbitMQ

**Comment by MrFabio** [comment]
Pub/Sub

**Comment by kiwicopple** [comment]
Webhook
